guacamole/src/main/webapp/translations/ur.json

```diff
@@ -0,0 +1,874 @@
+{
+    
+    "NAME" : "اردو",
+    
+    "DIRECTION":"rtl",
```

Comment:
Please follow the style of the surrounding code (see other comment). The new JSON should be consistent in style with existing translations.